solarforecastarbiter/reports/main.py

```diff
@@ -155,6 +156,36 @@ def get_validation_issues():
     return test
 
 
+def validate_resample_align(report, metadata, data):
+    """
+    Validate the data and resample.
+
+    Parameters
+    ----------
+    report : solarforecastarbiter.datamodel.Report
+    metadata : solarforecastarbiter.datamodel.ReportMetadata
+    data : dict
+        Keys are Forecast and Observation uuids, values are
+        the corresponding data.
+
+    Returns
+    -------
+    list
+        List of solarforecastarbiter.datamodel.ProcessedForecastObservation
+
+    Todo
+    ----
+    * Move to metrics.preprocessing? Since it calls report and metadata I have
```

Comment:
I like that the metrics module code is independent of the report object, so I think it makes sense to keep it here.

Reply:
Ok. Removed the Todo.